### PR TITLE
[eclipse/xtext#1412] build against 2019-03 release

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1412] build against 2019-03 release
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel/src/org/eclipse/xtext/example/domainmodel/serializer/DomainmodelSyntacticSequencer.xtend [hunk 1]
```diff
@@ -3,6 +3,16 @@
  */
 package org.eclipse.xtext.example.domainmodel.serializer
 
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.RuleCall
+import org.eclipse.xtext.nodemodel.INode
 
 class DomainmodelSyntacticSequencer extends AbstractDomainmodelSyntacticSequencer {
+	
+	override protected String getArrayBracketsToken(EObject semanticObject, RuleCall ruleCall, INode node) {
```
Comment (kthoms) [inline org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel/src/org/eclipse/xtext/example/domainmodel/serializer/DomainmodelSyntacticSequencer.xtend]: why this change here?

Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel/src/org/eclipse/xtext/example/domainmodel/serializer/DomainmodelSyntacticSequencer.xtend]: (1) this is an older change that is due regeneration with the current mast.
(2) this change leads to [ ] be the default cause there are two keywords in the grammar
(3) this corrects this back

Comment (kthoms) [reply org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel/src/org/eclipse/xtext/example/domainmodel/serializer/DomainmodelSyntacticSequencer.xtend]: OK
